src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -709,6 +713,8 @@ case class JesBackend(actorSystem: ActorSystem)
   }
 
   override def findResumableExecutions(id: WorkflowId)(implicit ec: ExecutionContext): Future[Map[ExecutionDatabaseKey, JobKey]] = {
-    globalDataAccess.findResumableJesExecutions(id)
+    globalDataAccess.findResumableExecutions(id, IsResumable, BuildJobKey)
   }
+
+  override def executionInfoKeys: Seq[String] = Seq("JES_RUN_ID", "JES_STATUS")
```

Comment:
I still don't get it... so Seq creates a List... why can that lead to bugs?  It sounds like Thibault's problem is less to do with `Seq` versus `Vector` constructors and more to do with ordered versus unordered Iterables